api/v1/values_types_extra.go

```diff
@@ -90,8 +99,11 @@ type ZTunnelConfig struct {
 	XdsAddress *string `json:"xdsAddress,omitempty"`
 	// Specifies the default namespace for the Istio control plane components.
 	IstioNamespace *string `json:"istioNamespace,omitempty"`
-	// Same as `global.logging.level`, but will override it if set
-	Logging *GlobalLoggingConfig `json:"logging,omitempty"`
```

Comment:
We can't remove fields from the API.

Reply:
Shall I mark it as deprecated?

Reply:
Oh, wait. The `ZTunnel` resource is still `v1alpha1`, right? So, we *can* remove it.